src/main/java/net/torocraft/signedit/SignEdit.java

```diff
@@ -60,12 +68,9 @@ public void editSign(RightClickBlock event) {
     }
   }
 
-  private static boolean isHoldingSign(EntityPlayer player) {
-    for (ItemStack stack : player.getHeldEquipment()) {
-      if (stack.getItem() == Items.SIGN) {
-        return true;
-      }
-    }
+  private static boolean isHoldingEditor(EntityPlayer player) {
+    if (editor == null) return true;
+    for (ItemStack stack : player.getHeldEquipment()) if (stack.getItem() == editor) return true;
```

Comment:
use the multi-line style that was use originally, again no need to change code style for this